src/eth_provider/error.rs

```diff
@@ -361,13 +367,13 @@ mod tests {
     #[test]
     fn test_display_execution_error() {
         // Given
-        let err = EthApiError::Kakarot(KakarotError::Execution(ExecutionError::Evm(EvmError::BalanceError)));
```

Comment:
I'm not sure why you changed this?

Reply:
I changed to test if the errors i've created would pass the test, and they did. But I can change back for the balance test, if that is what you guys want.

Reply:
changed back.